functional/vfio/run.sh

```diff
@@ -20,7 +20,16 @@ tmp_data_dir="$(mktemp -d)"
 trap cleanup EXIT
 
 CONFIG_FILE="${tmp_data_dir}/configuration.toml"
+
+# kata-runtime options
 SANDBOX_CGROUP_ONLY=""
+hypervisor=
+machine_type=
+
+# config option paths and strings
+config_options_path="/usr/share/defaults/kata-containers"
+hypervisor_clh="configuration-clh.toml"
+hypervisor_qemu="configuration-qemu.toml"
```

Comment:
please move these two to `setup_configuration_file` and change their name. `clh_config_file` and `qemu_config_file` looks better ? wdyt?

Reply:
Yes those names look better. To be sure, you're suggesting I move them to setup_configuration_file and have the strings hard coded there?

Reply:
yes, please